lib/mutant/repository.rb

```diff
@@ -61,6 +63,21 @@ def touches?(path, line_range)
         !stdout.empty?
       end
 
+    private
+
+      # Test if path is tracked in repository
+      #
+      # @param [Pathname] path
+      #
+      # @return [Boolean]
+      #
+      # @api private
+      def tracks?(path)
+        command = %W[git ls-files --error-unmatch -- #{path}]
+        _stdout, _stderr, status = Open3.capture3(*command, binmode: true)
+        status.success?
```

Comment:
The other method has 2 branches. This one not.

I added the blank lines in the other method to make sure the branches are not "buried" in the code.